src/plugins/xerces/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+include (LibAddMacros)
+
+if (DEPENDENCY_PHASE)
+	find_package(XercesC)
+
+	if (NOT XercesC_FOUND)
+		remove_plugin (xerces "XercesC library not found")
```

Comment:
i tested it with 3.0.0, worked, so i use 3.0.0 as a minimum version. Version 2 is no longer maintained, so i think its good to avoid that version (its from 2008). Maybe it works though, i haven't checked, but can do if you think it makes sense.